### PR TITLE
feat: Add enable authzMode option

### PR DESCRIPTION
**Pull request by Hemanthghs**

### CODE REVIEW

File: frontend/src/pages/gov/ActiveProposals.js [hunk 1]
```diff
@@ -1,20 +1,49 @@
...
+          alignItems: "center",
+          justifyContent: "flex-end",
+        }}
+      >
+        <Typography sx={{ fontWeight: "500", color: "text.primary" }}>
+          Authz
+        </Typography>
+        <Switch checked={checked} onChange={switchHandler} />
+      </Box>
+      {chains.map((key, index) => (
```
Comment (charymalloju) [inline frontend/src/pages/gov/ActiveProposals.js:38]: whenever we do mapping in the react. It should have parent tags separately or at least we have to use react fragment tags. i.e, <> your code </> or <React.Fragment> your code </React.Fragment/>

File: frontend/src/pages/gov/Proposals.js [hunk 2]
```diff
@@ -46,13 +48,38 @@ export default function Proposals({ chainUrl, chainName, chainLogo }) {
...
       const response = async (chainUrl) => {
         const res = await govService.proposals(chainUrl);
         return res.data;
       };
-      response(chainUrl).then((res) => {setProposals(res.proposals)});
+      response(chainUrl).then((res) => {
+        setProposals(res.proposals);
+      });
+
+      const grantsToMeResponse = async (data) => {
```
Comment (charymalloju) [inline frontend/src/pages/gov/Proposals.js:62]: we have to use try and catch block whenever we use async-await.
```suggestion
      const grantsToMeResponse = async (data) => {
           try {
              const response = await service.grantsToMe(
                      data.baseURL,
                      data.grantee,
                      data.pagination
                    );
                    return response.data;
           } catch (e){
             throw e
           }
      }
```

File: frontend/src/pages/gov/Proposals.js [hunk 3]
```diff
@@ -46,13 +48,38 @@ export default function Proposals({ chainUrl, chainName, chainLogo }) {
...
+          data.grantee,
+          data.pagination
+        );
+        return response.data;
+      };
+      grantsToMeResponse({
+        baseURL: chainUrl,
+        grantee: address,
+      }).then((res) => {
+        if (res.grants.length > 0) {
```
Comment (charymalloju) [inline frontend/src/pages/gov/Proposals.js:74]: Always have to check attributes in the object  res?.grants?.length

File: frontend/src/pages/gov/Proposals.js [hunk 4]
```diff
@@ -46,13 +48,38 @@ export default function Proposals({ chainUrl, chainName, chainLogo }) {
...
+          if (
+            res.grants[0].authorization.msg === "/cosmos.gov.v1beta1.MsgVote"
```
Comment (charymalloju) [inline frontend/src/pages/gov/Proposals.js:76]: In most of the places we use msg names so define them in the constant file.

Comment (aleem1314) [inline frontend/src/pages/gov/Proposals.js:76]: msg can be one of `"/cosmos.gov.v1beta1.MsgVote"` or `"/cosmos.gov.v1.MsgVote".

**Pull request merged**